packages/grid/_modules_/grid/utils/tree/sortRowTree.ts

```diff
@@ -0,0 +1,76 @@
+import {
+  GridFieldComparatorList,
+  GridRowId,
+  GridRowTreeConfig,
+  GridRowTreeNodeConfig,
+} from '../../models';
+
+interface SortRowTreeParams {
+  rowIds: GridRowId[];
+  rowTree: GridRowTreeConfig;
+  disableChildrenSorting: boolean;
+  comparatorList: GridFieldComparatorList;
+  sortRowList: (rowList: GridRowTreeNodeConfig[]) => GridRowId[];
+}
+
+export const sortRowTree = (params: SortRowTreeParams) => {
```

Comment:
Copy / paste from `useGridSorting`
The behavior is the same in `useGridGroupingColumns` so I set this method in an util folder to use it for both features.